src/components/portal/pages/PortalPageBase.tsx

```diff
@@ -10,20 +10,26 @@ import {contentStyle} from 'styles/base';
 const breadcrumb:string = css`
     top: 95px !important;
     position: relative;
-    left: 20px !important; 
+    left: 20px !important;
   `;
 
 export default abstract class PortalPageBase extends PageBase {
   abstract renderContent();
 
+  renderBreadcrumb() {
```

Comment:
Is looking good.
Main thing that I would add is to add documentation to the renderBreadcrumb function stating:
1. That it can (And should) be overwritten for pages which want a customized breadcrumb navigation
2. A brief example as to how to best overwrite. Simply pointing to the implementation in ReviewsHome should suffice, but we don't want to be scrambling to recall when overwriting in the future.